### ISSUE TITLE
A [PS VR Adaptor] Case

### ISSUE THREAD

**Issue by saradasama** [psxdev/OrbisEyeCam#7]
I had the opportunity to test 2 ‘PS Camera V2’ models (manufactured in September 2018 and March 2019) with 1 ‘PS VR Camera Adaptor for PS5’ (manufactured in December 2020, which is probably from the first wave of this product).

I tested a total of 10 different healthy [USB 3.0] ports on 2 computers (without USB extension cable), and at least 7 of the ports were Intel-based [3.0] ports with their latest ‘eXtensible Host Controller’ drivers (on [Z77] and [X99] chipsets). The [Windows 10] operating systems were fully up-to-date and they operate flawlessly.

The issue was the same everywhere: the camera devices were successfully transformed into the ‘~OV 580’ camera device (without any software error), but the screen was always black (I was using multiple applications for checking the screen content and changing the various properties). The possible selectable resolutions and frame rates were available but nothing was displayed.

I think the cause of this issue can be originated from the communication via the ‘PS VR Camera Adaptor’ — this was the only factor which could not have been changed during the tests, so the adaptor might be the culprit. **Would the developers thoroughly investigate an official PS5 adaptor for possible hardware or driver inconsistencies?**

**Comment by Hackinside** [comment]
Did you try using an unofficial PS AUX to USB 3.0 adapter?

**Comment by saradasama** [comment]
> Did you try using an unofficial PS AUX to USB 3.0 adapter?

No, I used the official ‘PS VR Camera Adaptor’. But I know about the unofficial device, so I ask now: how can I contact you directly to buy one?

**Comment by Hackinside** [comment]
Via discord Hackinside#7031
